README.md

```diff
@@ -367,6 +367,12 @@ the adjust backend will convert it to a universal link, which looks like this:
 https://[hash].ulink.adjust.com/ulink/path/?key=foo&value=bar
 ```
 
+We provide an helper function that allows to convert the universal link to a deeplink url. 
```

Comment:
`We provide a helper function that allows you to convert a universal link to a deeplink url.`